src/generateContent.js

```diff
@@ -4,10 +4,11 @@ import DEFAULT_OPTIONS from './defaultOptions';
  * @param {String} module
  * @param {String} locale
  * @param {Object} translations
+ * @param {Boolean} mustRequireAngular
  * @return {String}
  */
-export default (module = DEFAULT_OPTIONS.module, locale = DEFAULT_OPTIONS.defaultLocale, translations = {}) => {
-    return `var angular = require("angular");
+export default (module = DEFAULT_OPTIONS.module, locale = DEFAULT_OPTIONS.defaultLocale, translations = {}, mustRequireAngular = DEFAULT_OPTIONS.requireAngular) => {
+    return `${mustRequireAngular ? 'var angular = require("angular");' : ''}
```

Comment:
@Ardeshir81 We need to do something like that https://github.com/WearyMonkey/ngtemplate-loader/blob/master/index.js#L58

Reply:
smarter ...